client/client.go

```diff
@@ -91,14 +94,16 @@ func handleResp(req *http.Request) (respBodyObj ResponseBody, err error) {
 		_ = Body.Close()
 	}(resp.Body)
 	respBody, err := io.ReadAll(resp.Body)
+
+	log.WithFields(log.Fields{
```

Comment:
Do we plan to leave this here for the debugging purposes or would this be displayed to the user ?